project/Newevaluation.html

```diff
@@ -6,85 +6,118 @@
   <style>
     body{
       background-color: rgb(82, 86, 89);
-      margin-top: 50px;
+      margin-top: 3%;
       margin-left: 0;
     }
 
+
+
+/* ---------------------------------Navigation Bar--------------------------- */
+
     .active{
       border-bottom: 6px solid DodgerBlue;
-      padding:18px;
+      padding: 1%;
```

Comment:
Because that looked good on my screen I guess. I know on a mobile screen 1% means almost nothing but I thought that using later (today I hope) media queries I can add a breakpoint and change there the padding so it will be suitable for a smaller screen.